homeassistant/components/tibber/sensor.py

```diff
@@ -146,13 +152,25 @@ def _update_current_price(self):
             if now.date() == price_time.date():
                 max_price = max(max_price, price_total)
                 min_price = min(min_price, price_total)
+                if price_time.hour < 8:
+                    off_peak_1 = max(off_peak_1, price_total)
+                    num1 += 1
+                elif price_time.hour < 20:
+                    off_peak = max(off_peak, price_total)
+                    num0 += 1
+                else:
+                    off_peak_2 = max(off_peak_2, price_total)
+                    num2 += 1
```

Comment:
What does this logic do? Why doesn't it belong in the tibber library?

Reply:
They are values reported by Norpool.

> 
> Peak Peak refers to the average price of the period from 8am to 8pm. Off peak 1 refers
> to the average price of the time period from midnight to 8am and off peak 2 refers
> to the average price of the time period from 8pm to midnight.
> https://www.energiforetagen.se/globalassets/energiforetagen/det-erbjuder-vi/kurser-och-konferenser/krisutbildningar/nord-pool-spot-glossary.pdf

I can move it to `pyTibber`, but then I should probably also move the max, min and average calculations?

Reply:
If this calculation is always or often necessary to make use of these values, I would put it in the library.